BUILDING.md

```diff
@@ -9,15 +9,24 @@ If you consistently can reproduce a test failure, search for it in the
 file a new issue.
 
 
-### Unix / Macintosh
+### Unix / OS X
 
 Prerequisites:
 
 * `gcc` and `g++` 4.8 or newer, or
 * `clang` and `clang++` 3.4 or newer
 * Python 2.6 or 2.7
 * GNU Make 3.81 or newer
-* libexecinfo (FreeBSD and OpenBSD only)
+
+On OS X, you will also need:
+* [Xcode](https://developer.apple.com/xcode/download/)
+  * You also need to install the `Command Line Tools` via Xcode. You can find
```

Comment:
Unfortunately, to test this, we need a pristine OS X for someone to try it out on. Anyone? Anyone?

Does the OS X in CI build from scratch, more or less? If so, does it install XCode?